docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
     image: mariadb:10.5.9
     env_file:
       - .env
+    ports:
+      - "3306:3306"
```

Comment:
Is there a way we could use this port mapping just in development mode? Asking that because I believe that when @pxotox first created it he was thinking of having the `db` container being accessible only to `app`, internally. By adding this line, we're exposing the database to the host instance as well (although it will have another protection layer with the security groups' port firewall). To be discussed, not a problem for now.

Reply:
I just commited this because everytime I use the command "git add ." it was added and I'm tired to unstage.
I removed the change.

But answering your question, the way I know to do this is creating two yml (prod/dev) files and load the correct one according current environment.

Reply:
It would be a good idea to create a separate task to define the best approach.

Reply:
Sure, I added a comment about it on #4 too.